.changeset/beauty-die-joy.md

```diff
@@ -0,0 +1 @@
+TODO: delete me
```

Comment:
```suggestion
---
---

```

Reply:
This came from the precociously-deleted `rc: beetle`.

This file can be removed entirely.

Reply:
I'll let this beauty die joyfully, then.